wallet/res/layout/dialog_tax_category_explainer.xml

```diff
@@ -225,17 +236,20 @@
 
         <TextView
             android:id="@+id/transfer_in_text"
-            style="@style/Caption"
+            style="@style/Caption.Tertiary"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="5dp"
-            android:background="@color/steel_gray_40"
+            android:paddingHorizontal="5dp"
+            android:background="@drawable/rounded_background"
+            android:theme="@style/TertiaryTextHighlight"
             tools:text="Transfer-in"
-            android:textColor="@color/light_dark_gray"
             android:textSize="13sp"
             app:layout_constraintBottom_toBottomOf="@+id/you_can_change_it_two"
             app:layout_constraintStart_toEndOf="@+id/you_can_change_it_two"
-            app:layout_constraintTop_toTopOf="@+id/you_can_change_it_two" />
+            app:layout_constraintTop_toTopOf="@+id/you_can_change_it_two"
+            app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Unfortunately, even with an end constraint, these stacked texts can get cut off since they aren't wrapping automatically. 
This can be a problem with different translations especially.
We could probably solve this by using `SpannableText` on a single TextView or by using `GridLayout` to make views wrap themselves.

For now, I've only decreased horizontal margins on the parent. The content seemed a bit too packed to the center anyway - it's only `28dp` in the design while here it was `30dp` + `8dp` for the icon padding

<img src="https://user-images.githubusercontent.com/7733989/175265487-3f2bf071-5a4b-465c-bd07-12efd38af642.jpg" width="400" />